docs/security/principles/01-locked-vault-is-secure.mdx

```diff
@@ -0,0 +1,13 @@
+# P01 - A locked vault is secure
+
+Clients must ensure that once the vault has been locked, no vault data can be accessed in plain
+text, even if the device becomes compromised after the lock occurs. Protections are not guaranteed
```

Comment:
What about vault data that is intentionally stored in plain text? e.g. modification dates, folder guids, boolean values. Or is this excluded by the definition of "vault data"? (is metadata "sensitive and private information"?)

Reply:
Yes, there is actually still an open thread in confluence about this because I was hoping to get Vault involved in classifying data a bit better. My idea is to split up the vault data into groups of varying sensitivity. In that case we might get:

- High: Password
- Moderate: Username
- Low: Creation date

And then say something like "low sensitivity data" may be stored unprotected

Reply:
I think that would work!

Reply:
I tweak this a little bit by only grouping the data into highly sensitive and less sensitive (see 528330f) to avoid getting into hard discussions. What do you think?